src/supermarket/spec/models/cookbook_spec.rb

```diff
@@ -725,16 +725,19 @@ def generate_params_versions(opts = {})
   describe ".ordered_by" do
     let!(:great) { create(:cookbook, name: "great") }
     let!(:cookbook) { create(:cookbook, name: "cookbook") }
+    let!(:deprecated_cookbook) { create(:cookbook, name: "deprecated_cookbook", deprecated: "true") }
 
     it "orders by name ascending by default" do
-      expect(Cookbook.ordered_by(nil).map(&:name)).to eql(%w{cookbook great})
+      expect(Cookbook.ordered_by(nil).map(&:name)).to eql(%w{cookbook great deprecated_cookbook})
     end
 
     it 'orders by updated_at descending when given "recently_updated"' do
       great.touch
 
+      puts "dep cook book updated at #{deprecated_cookbook.updated_at} #{deprecated_cookbook.deprecated}"
+      puts " cookbook updated_at #{cookbook.updated_at} #{cookbook.deprecated}"
```

Comment:
@msys-sgarg You might consider removing these "puts" statements from here in tests.